packages/ui/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact ui module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `ui/VirtualList.VirtualList` and `ui/VirtualList.VirtualGridList` not to scroll while pressing 5-way keys after wheeling.
```

Comment:
How about "not to suddenly jump when pressing directional keys after wheeling"?

Reply:
Fix it as you mentioned.